LiteFarmOrg/LiteFarm
# LF-2658 location creation popup button text fix

This PR is a simple capitalization fix for button text.

To Test:
1. Have a farm with no locations.
2. Try creating a task.
3. The popup should have two buttons: "Go back" and "Create location".
